docs/tutorials/aws.md

```diff
@@ -102,11 +102,13 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: external-dns
+  namespace: default
 ---
 apiVersion: rbac.authorization.k8s.io/v1beta1
 kind: ClusterRole
 metadata:
   name: external-dns
+  namespace: default
```

Comment:
A `ClusterRole` is a resource that is not namespaced, so it is not correct to specify a namespace, see the [example from the official doc](https://kubernetes.io/docs/reference/access-authn-authz/rbac/): 

```
kind: ClusterRole
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  # "namespace" omitted since ClusterRoles are not namespaced
  name: secret-reader
rules:
- apiGroups: [""]
  resources: ["secrets"]
  verbs: ["get", "watch", "list"]
```